Source/WebCore/accessibility/AccessibilityObject.cpp

```diff
@@ -627,14 +627,10 @@ static bool isTableComponent(AXCoreObject& axObject)
 
 void AccessibilityObject::insertChild(AXCoreObject* newChild, unsigned index, DescendIfIgnored descendIfIgnored)
 {
-    if (!newChild)
-        return;
-
-    ASSERT(is<AccessibilityObject>(newChild));
```

Comment:
Please add back this assertion. dynamicDowncast<>() (unlike downcast<>()) has no such assertion, so it is still useful

Reply:
Maybe we can just use `ASSERT(child);` below.